### PR TITLE
[MM-14877] add description above post:all

### PR DESCRIPTION
**Pull request by liusy182**
#### Summary

Wide screen:

Screen Shot 2019-04-17 at 11 32 56 PM

Narrow screen:

Screen Shot 2019-04-17 at 11 33 10 PM

#### Ticket Link

https://github.com/mattermost/mattermost-server/issues/10639

#### Related Pull Requests

- Has server changes (please link here)
- Has redux changes (please link here)
- Has mobile changes (please link here)

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks @liusy182 for the quick PR! Can you help resolve the build issue and I'll spin up a test server for us to review?

**Comment by liusy182** [comment]
seems to be working after latest fix :) @jasonblais

**Comment by jasonblais** [comment]
Hey @liusy182, can you help rebase against master? Looks like the second spinmint didn't get generated because of it.

Also, sorry for the delay in reviewing, this slipped my radar last week.

File: components/integrations/bots/add_bot/add_bot.jsx [hunk 1]
```diff
@@ -569,6 +570,14 @@ export default class AddBot extends React.Component {
...
                             </div>
                         </div>
+                        <div className='row bot-profile__section'>
+                            <div className='col-md-5 col-sm-8 col-sm-offset-4'>
+                                <FormattedMarkdownMessage
+                                    id='admin.manage_roles.additionalRoles'
+                                    defaultMessage='Select additional permissions for the account. [Read more about roles and permissions](!https://about.mattermost.com/default-permissions).'
+                                />
+                            </div>
+                        </div>
```
Comment (saturninoabril) [inline components/integrations/bots/add_bot/add_bot.jsx]: Can you please add test for this?

**Comment by liusy182** [comment]
i am not sure what would have resulted to the CI failure. Any idea?

**Comment by hanzei** [comment]
@jasonblais Can this be merged?

**Comment by jasonblais** [comment]
@hanzei Indeed. Thank you @liusy182 🎉

**Pull request merged**